data/yam/agama/auto/lib/scripts_post.libsonnet

```diff
@@ -6,5 +6,13 @@
       #!/usr/bin/env bash
       echo 'PermitRootLogin yes' > /etc/ssh/sshd_config.d/root.conf
     |||
+  },
+  enable_gdm: {
+  name: 'enable gdm login',
+  chroot: true,
+  content: |||
+    #!/usr/bin/env bash
+    systemctl enable gdm
+  |||
```

Comment:
you can save the workaround in the ticket, but actually it is expected to be fixed in next build, otherwise we should file a bug.

Reply:
OK, I'll save it in the ticket and drop it after run the VRs.